models/Project.model.ts

```diff
@@ -0,0 +1,17 @@
+import { DeletedTask } from "./DeletedTask.model";
+import { Field } from "./Field.model";
+import { Task } from "./Task.model";
+import { User } from "./User.model";
+
+export type Project = {
+  id: string;
+  title: string;
+  creator: User;
+  tasks: Task[];
+  members: User[];
+  creationDate: number;
+  deletedTask: DeletedTask[];
+  status: Field<Project>[];
+  priority: Field<Project>[];
```

Comment:
please delete and create separate file with default values as const
`export const taskPrisOptions: Field[] = { name: "status", option... }`